fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/panel/panel.cfg

```diff
@@ -143,6 +143,14 @@ background_color = 0,0,0
 
 htmlgauge00 = WasmInstrument/WasmInstrument.html?wasm_module=flypad-backend.wasm&wasm_gauge=FlyPadBackend,0,0,1,1
 
+[VCockpit19]
+size_mm = 0,0
+pixel_size = 0,0
+texture = $PFD
```

Comment:
As per [https://docs.flightsimulator.com/flighting/html/Content_Configuration/SimObjects/Aircraft_SimO/Instruments/panel_cfg.htm#invisible](https://docs.flightsimulator.com/flighting/html/Content_Configuration/SimObjects/Aircraft_SimO/Instruments/panel_cfg.htm#invisible) you can use `NO_TEXTURE` (without dollar sign `$`) to make an invisible gauge and therefore potentially save resources.

Maybe we could make `systems-host` do this as well while we're here

Reply:
A good - didn't know.
Yes - I can change the systems-host as well - the PR is rebased